lang/src/org/partiql/lang/planner/GlobalBindings.kt

```diff
@@ -0,0 +1,50 @@
+package org.partiql.lang.planner
+
+import org.partiql.lang.eval.BindingCase
+import org.partiql.lang.eval.BindingName
+
+/** Indicates the result of an attempt to resolve a global binding. */
```

Comment:
I understand we have kdoc for `GlobalBindings.resolve()` but can we also add a definition for _Global Binding_ here?

BTW. 
I think it's helpful to have the link to the upcoming documentation in applicable constructs under `planner`.

Reply:
I have rephrased this to be more clear.